src/libraries/System.Private.CoreLib/src/Resources/Strings.resx

```diff
@@ -3688,4 +3688,7 @@
   <data name="NotSupported_MethodBodyReplacement" xml:space="preserve">
     <value>Method body replacement not supported in this runtime.</value>
   </data>
+  <data name="NotSupported_COM" xml:space="preserve">
+    <value>Built-in COM has been disabled via a feature switch</value>
```

Comment:
Do we have some sort of aka.ms link for feature switches? I see above that `ResourceManager_ReflectionNotAllowed` specifically calls out an MSBuild property, but if we're continuing to add switches that can result in user-visible exceptions, it might be nice to just have a link to doc for all feature switches.